initramfs/init.sh

```diff
@@ -126,8 +135,8 @@ find_and_run_update()
 			break;
 		fi
 
-		echo "Found '${SYSTEM_UPDATE_ENTRYPOINT}' script on ${dev}, trying to execute."
-		if ! "${SYSTEM_UPDATE_ENTRYPOINT}"; then
+		echo "Found '${SYSTEM_UPDATE_ENTRYPOINT}' script on ${TOOLBOX_MOUNT}, trying to execute."
```

Comment:
Not sure if the "on TOOLBOX_MOUNT" really adds anything, just like the "on dev" didn't really tell you much (as it was already known that it was mounted from previous echo lines, and this thus tells the mount point, which doesn't really tell much)

Maybe it should just be "Found ENTRYPOINT script in toolbox, trying to execute."

Reply:
Fixed, unfortunately have to approve again.